renku/core/commands/format/dataset_files.py

```diff
@@ -52,8 +57,33 @@ def jsonld(client, records):
     return dumps(data, indent=2)
 
 
+def _make_headers(columns):
```

Comment:
This method is identical to the `_make_headers` for `DatasetFile` except for `DATASET_COLUMNS` vs `DATASET_FILES_COLUMNS`. We can have a generic `make_headers` method that takes `DATASET_COLUMNS`/`DATASET_FILES_COLUMNS` as additional input parameter to reduce code duplication.

Reply:
I've actually factored out `tabular` function which also has a similar pattern in both files.